src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildAdapter.kt

```diff
@@ -52,6 +54,15 @@ class CargoBuildAdapter(
             .withExecutionFilters(*createFilters(context.cargoProject).toTypedArray())
             .withRestartAction(createRerunAction(processHandler, context.environment))
             .withRestartAction(createStopAction(processHandler))
+            .withContentDescriptorSupplier {
```

Comment:
Can avoid using the deprecated methods here?
Looks like we can use the same API as we use for Sync tool window (see `org.rust.cargo.project.model.impl.CargoSyncTask#createSyncProgressDescriptor`)